### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by CalvinScott**
Update and improve the README.

### CODE REVIEW

**Comment by CalvinScott** [comment]
@CMClay @slajax Please review whenever you're free

File: Readme.md [hunk 1]
```diff
@@ -17,28 +19,37 @@ Include the hosted SDK in your HTML header:
...
 ```
 
-#### Use with component
+2. Next, in any of your files used to compile your client-side application, make sure to insert the dependency with:
+
+```js
+var retsly = require('retsly-js-sdk');
+```
+
+Now you are fully qualified to make use of this component of our API.
```
Comment (slajax) [inline Readme.md]: `Now you will be able to use this module to interact with our API.`

File: Readme.md [hunk 2]
```diff
@@ -17,28 +19,37 @@ Include the hosted SDK in your HTML header:
...
-Use with `require`:
+4. Finally, drop a single `<script>` tag into your html file:
 
-```js
-var Retsly = require('retsly-js-sdk')
+```html
+<script src="target.js"></script>
 ```
 
+Now you should be able to make use of the utilities within `retsly-js-sdk`
```
Comment (slajax) [inline Readme.md]: `Now you should be able to make use of all of the Retsly API endpoints via the `retsly-js-sdk`

**Comment by slajax** [comment]
Good job @CalvinScott, few little text updates then :+1:

**Comment by CMClay** [comment]
:+1:

**Pull request merged**